src/components/Table/Table.jsx

```diff
@@ -458,10 +459,16 @@ const Table = React.createClass(createLucidComponentDefinition({
 		 * Adjusts the style of the table to have more spacing within the table cells
 		 */
 		hasExtraWhitespace: bool,
+
+		/**
+		 * render the table without borders on the outer edge
+		 */
+		hasNoBorder: bool,
```

Comment:
is there a reason for this to be `hasNoBorder` with default `false` instead of `hasBorder` with default `true`?